libvast/vast/system/actors.hpp

```diff
@@ -519,7 +527,8 @@ CAF_END_TYPE_ID_BLOCK(vast_actors)
 // never be sent over the network.
 #define vast_uuid_synopsis_map std::map<vast::uuid, vast::partition_synopsis>
 CAF_ALLOW_UNSAFE_MESSAGE_TYPE(std::shared_ptr<vast_uuid_synopsis_map>)
-CAF_ALLOW_UNSAFE_MESSAGE_TYPE(std::shared_ptr<vast::partition_synopsis>)
+CAF_ALLOW_UNSAFE_MESSAGE_TYPE(vast::partition_synopsis_ptr)
```

Comment:
I don't think this needs to be unsafe anymore now that it is an intrusive pointer.

Reply:
Yes. (and it makes sense to me, why should caf magically know how to serialize a type just because it's wrapped in an intrusive pointer)

```
/home/benno/src/vast/transforms/libvast/aux/caf/libcaf_core/caf/make_message.hpp:83:74: note: ‘caf::detail::tl_forall<caf::detail::type_list<caf::intrusive_cow_ptr<vast::partition_synopsis> >, caf::is_serializable_or_whitelisted>::value’ evaluates to false
/home/benno/src/vast/transforms/libvast/aux/caf/libcaf_core/caf/make_message.hpp:92:55: error: static assertion failed: at least one type has no type ID: please assign type IDs to all of your types via CAF_ADD_TYPE_ID (this check was enabled via CAF_ENABLE_TYPE_ID_CHECKS)
   92 |   static_assert(tl_forall<stored_types, has_type_id>::value,
```

Reply:
Ah, so that is missing as well. I've written down a note to write an inspector for both partition synopsis and partition synopsis pair sometime in the future, we need this for the CAF 0.18 upgrade.